app/Console/Commands/ExternalServices/ManageSlack.php

```diff
@@ -121,4 +139,14 @@ private function messageUserAdvisingOfRegistration($slackUser)
         $this->sendSlackMessagePlain($slackUser->id, 'If you have problems with this, please get in touch: https://helpdesk.vatsim.uk', 'VATSIM UK Slack Bot');
         $this->sendSlackMessagePlain($slackUser->id, '****************************************************', 'VATSIM UK Slack Bot');
     }
+
+    private function messageUserAdvisingOfAutomaticRegistration($slackUser)
```

Comment:
Agreed. I'll change this for them all.

Reply:
Perhaps this could be extended to the other messages being sent in this class? Are there any areas of the application that Slack message are sent from?

Reply:
Agreed. I think that could be discussed at a later date, possibly through the abstraction of that functionality. My original thought with passing the array is that each element of the array would be a new line, but I think your implementation definitely achieves what I was getting at in the first place.

Reply:
Agreed that what we have here is a good solution for now and we can refactor further at a later date if required.